src/zimlet/com_zimbra_attachmail/attachmail.css

```diff
@@ -28,35 +28,136 @@
 }
 
 .AttachMailTabBox {
-    background-color: #FDFCFA;
-    border-right:1px solid #E0E0E0;
+    background-color: #ffffff;
```

Comment:
I think, yes, we can't use skin.properties variables here.
@tarangkhandelwal , let me know if I am wrong.